storage/src/cache/dummycache.rs

```diff
@@ -149,9 +149,17 @@ impl BlobCache for DummyCache {
             }
         }
 
-        copyv(&buffer_holder, bufs, offset as usize, user_size, 0, 0)
-            .map(|(n, _)| n)
-            .map_err(|e| eother!(e))
+        copyv(
+            &buffer_holder,
+            bufs,
+            offset as usize,
+            // Even a merged IO can hardly reach u32::MAX. So this is safe
```

Comment:
Seems this comment is for line 148? It's always safe to cast u32 to usize.

Reply:
Yes. Let's move it to L148